po/ru.po

```diff
@@ -549,15 +545,20 @@ msgstr "Повернуть влево"
 msgid "Rotate Right"
 msgstr "Повернуть вправо"
 
-#: data/menu.ui:152 data/menu.ui:420
+#: data/menu.ui:152 data/menu.ui:427
 msgid "Rotate _Left"
 msgstr "Повернуть в_лево"
 
 #: pdfarranger/pageutils.py:466 pdfarranger/splitter.py:60
 msgid "Rows"
 msgstr "Ряды"
 
-#: data/menu.ui:232 data/menu.ui:401
+#: pdfarranger/pdfarranger.py:3070
+#, python-format
+msgid "Running on %s"
```

Comment:
Left this empty on purpose? The text will be in about dialog. %s is replaced (at runtime) with name of operating system.

Reply:
fixed